contrib/fwupd.spec.in

```diff
@@ -454,7 +454,9 @@ done
 %endif
 %{_datadir}/installed-tests/fwupd/chassis_type
 %{_datadir}/installed-tests/fwupd/sys_vendor
+%if 0%{?fedora} >= 37
```

Comment:
This won't work well for rhel will it? Anything better that can be checked?

Reply:
Randomly, I'm looking at making the spec file work for RHEL 9 too next week. Will do some digging.